src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/spark/SparkExecutor.scala

```diff
@@ -92,17 +92,17 @@ object SparkExecutor extends Executor {
     sparkParameters.additionalJars.foreach(additionalJar => sparkLauncher.addJar(additionalJar))
     sparkParameters.additionalFiles.foreach(additionalFile => sparkLauncher.addFile(additionalFile))
     sparkParameters.additionalSparkConfig.foreach(conf => sparkLauncher.setConf(conf._1, conf._2))
-    mergeAdditionalSparkConfig(SparkExecutorConfig.getAdditionalConfs, sparkParameters.additionalSparkConfig, keysToMerge)
+    mergeAdditionalSparkConfig(SparkExecutorConfig.getAdditionalConfs, sparkParameters.additionalSparkConfig)
```

Comment:
Does this override configs from lines above ? 
    SparkExecutorConfig.getAdditionalConfs.foreach(conf => sparkLauncher.setConf(conf._1, conf._2))
and
    sparkParameters.additionalSparkConfig.foreach(conf => sparkLauncher.setConf(conf._1, conf._2))

Reply:
Yes, but it only overrides the configs that were merged. I could also filter out the `KeysToMerge` in `sparkParameters.additionalSparkConfig.foreach`, but that would mainly just be for better readability, the end result wouldn't change.

Reply:
Yes, I thought so. You can leave it as it is.